code/__HELPERS/~skyrat_helpers/unsorted.dm

```diff
@@ -13,3 +13,36 @@
 	M.Scale(squishx, squishy)
 	animate(src, transform = M, time = halftime, easing = BOUNCE_EASING)
 	animate(src, transform = OM, time = halftime, easing = BOUNCE_EASING)
+
+/** Get all hearers in range, ignores walls and such. Code stolen from `/proc/get_hearers_in_view()`
+ * Much faster and less expensive than range()
+*/
+/proc/get_hearers_in_range(range_radius, atom/source)
+	var/turf/center_turf = get_turf(source)
+	if(!center_turf)
+		return
+
+	. = list()
+
```

Comment:
```suggestion
	var/old_luminosity = center_turf.luminosity
```
Current code will not compile and will brick upon being TM'd, this will fix that

Reply:
Missed that, good oatch